boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+Overview
+========
+
+The Kernelspace Zion is a small generic development board, featuring an
+ARM Cortex-M4 STM32F303K8 MCU with the minal required hardware layout
+for beginners to get started quickly. It's a simple open-hardware board
+that has been thought to be assembled easily at home.
+
+Specifications, schematic and pcb layout are available at the related
+`Zion page <https://kernel-space.org/hardware/zion.html>`_
+
+In brief:
+
+* STM32 microcontroller in LQFP32 package
+* GPIO pads all around (JXX) for a generic/quick usage
+* I2C display
+* Board power supply: through USB connector, 5 V supply voltage
+* Reset push button
+* One generic LEDs
+* JTAG header
+* Optional external crystal
+
+.. image:: img/zion.webp
+     :align: center
+     :alt: Zion
+
+
+Hardware
+========
+
+- STM32F303K8 in LQFP32 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 72 MHz max CPU frequency
+- VDD 5.0V
+- 64 KB Flash
+- 12 KB SRAM
+- GPIO with external interrupt capability
+- 2x12-bit ADC
+- 3x12-bit D/A converters
+- 7 channel DMA controller
+- 22 PWM channels
+- SPI (1)
+- I2C (1)
+- USART (3)
+- CAN (1)
+- RTC
+- General Purpose Timers (5)
+- Infrared transmitter
+- I2C display
+
+
+Default peripheral mapping
+==========================
+.. rst-class:: rst-columns
+
+- XTAL : PF1
+- XTAL : PF0
+- J25 : PB3 : JTDO
+- J3 : PB0
+- J4 : PB1
+- J5 : PB4
+- J6 : PB5
+- J7 : PB6 : I2C_1 SCL
+- J8 : PB7 : I2C_1 SDA
+- J9 : PA0
+- J10 : PA1
+- J11 : PA2
+- J12 : PA3
+- J13 : PA4
+- J14 : PA5
+- J15 : PA6
+- MOSFET : PA7
+- J16 : PA8
+- J17 : PA9
+- J18 : PA10
+- J19 : PA11
+- LED : PA12
+- J25 : PA13 : JTMS
+- J25 : PA14 : JTCK
+- J25 : PA15 : JTDI
+
+
+System Clock
+============
+
+Oscillator is set by default as hsi, 8Mhz, and system clock set to 72Mhz
+(max fo this mcu).
+
+
+Building and programming
+========================
+
+Building an example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/blinky
+   :board: ks_zion
+   :goals: build
+
+
+Flashing:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/blinky
+   :board: ks_zion
+   :goals: flash
+
+
+Debugging
+=========
+
+You can debug an application in the usual way. Here is an example for the
+**blinky** application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/blinky
+   :board: ks_zion
+   :goals: debug
```

Comment:
Once again, please try to follow the same style as other boards (please look at the source of the doc files if you're not sure, using an ST boards as a reference, for example), using proper headings etc. 
Sorry if this sounds like a nit, but beyond providing consistency across various pages of the documentation, it also helps structure the document and provide useful metadata for search engines.